streams/dshr_strdata_mod.F90

```diff
@@ -1091,11 +1124,11 @@ subroutine shr_strdata_advance(sdat, ymd, tod, logunit, istr, timers, rc)
              call ESMF_TraceRegionEnter(trim(lstr)//trim(timname)//'_coszenC')
              call shr_tInterp_getCosz(coszen, sdat%model_lon, sdat%model_lat, ymdmod(ns), todmod, &
                   sdat%eccen, sdat%mvelpp, sdat%lambm0, sdat%obliqr, sdat%stream(ns)%calendar, &
-                  sdat%mainproc, sdat%stream(1)%logunit)
+                  mainproc, logout)
              call ESMF_TraceRegionExit(trim(lstr)//trim(timname)//'_coszenC')
-             if (debug > 0 .and. sdat%mainproc) then
+             if (debug > 0 .and. mainproc) then
                 do n = 1,size(coszen)
-                   write(sdat%stream(1)%logunit,'(a,i4,2x,2(i18,2x),i8,d20.10)')' stream,ymdmod,todmod,n,coszen= ',&
+                   write(logout,'(a,i0,2x,2(i0,2x),i0,es13.6)')' stream,ymdmod,todmod,n,coszen= ',&
```

Comment:
I think you need another space before the `coszen`:
```suggestion
                   write(logout,'(a,i0,2x,3(i0,2x),es13.6)')' stream,ymdmod,todmod,n,coszen= ',&
```

Reply:
Fixed the format as follows:
```F90
 do n = 1,size(coszen)
       write(logout,'(2a,4(i0,2x),es13.6)') trim(subname), &
              ' stream,ymdmod,todmod,n,coszen= ',ns, ymd, tod, n, coszen(n)
 end do

```